### PR TITLE
openzone: init at 1.2.9

### LINKED ISSUE #93403: OpenZone cursor themes
**Project description**

https://www.gnome-look.org/p/999999/

**Metadata**

* homepage URL: https://github.com/ducakar/openzone-cursors
* source URL: https://github.com/ducakar/openzone-cursors
* license: MIT
* platforms: Linux

### PR DESCRIPTION
**Pull request by zaninime**
###### Motivation for this change

Fixes #93403. I'm currently using it on my desktop.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/openzone/default.nix
Comment (laikq) [inline pkgs/data/themes/openzone/default.nix:9]: I can't find a tag `v1.2.9` in the openzone repository, and `nix-build` also fails for me for that reason.

Comment (zaninime) [reply pkgs/data/themes/openzone/default.nix:9]: Something has gone wrong with renaming the package. I will fix it in a second.

**Comment by zaninime** [comment]
@laikq, mind giving a look again? Thanks.

**Comment by laikq** [comment]
From my perspective, it's looking good. I could install the cursor theme with nix-env, and changed it with lxappearance. Worked. But I think it would be a good idea to split this change into 2 commits, one for icon-slicer and one for openzone-cursors. (And I don't have merge/commit rights, so this PR needs another reviewer)

**Comment by zaninime** [comment]
I'm splitting the change in two commits for the two packages I'm adding.

**Comment by zaninime** [comment]
@dtzWill mind having a look as well and merge if appropriate?

File: pkgs/data/themes/openzone/default.nix
Comment (SuperSandro2000) [inline pkgs/data/themes/openzone/default.nix:21]:
```suggestion
    maintainers = with maintainers; [ zaninime ];
```

File: pkgs/tools/X11/icon-slicer/default.nix
Comment (SuperSandro2000) [inline pkgs/tools/X11/icon-slicer/default.nix:27]:
```suggestion
    maintainers = with maintainers; [ zaninime ];
```

File: pkgs/data/themes/openzone/default.nix
Comment (SuperSandro2000) [inline pkgs/data/themes/openzone/default.nix:22]: If icon slicer is linux only this can't be build on all platforms.

**Comment by zaninime** [comment]
@SuperSandro2000 could you please check again?

File: pkgs/tools/X11/icon-slicer/default.nix
Comment (SuperSandro2000) [inline pkgs/tools/X11/icon-slicer/default.nix:8]:
```suggestion
    url = "https://freedesktop.org/software/icon-slicer/releases/icon-slicer-${version}.tar.gz";
```

File: pkgs/tools/X11/icon-slicer/default.nix
Comment (SuperSandro2000) [inline pkgs/tools/X11/icon-slicer/default.nix:15]:
```suggestion
      url = "https://aur.archlinux.org/cgit/aur.git/plain/hotspotfix.patch?h=icon-slicer";
```

File: pkgs/tools/X11/icon-slicer/default.nix
Comment (SuperSandro2000) [inline pkgs/tools/X11/icon-slicer/default.nix:24]:
```suggestion
    description = "Utility for generating icon themes and libXcursor cursor themes";
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 93917` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>icon-slicer</li>
    <li>openzone-cursors</li>
  </ul>
</details>

**Comment by zaninime** [comment]
@SuperSandro2000 sorry for the extra delay. I applied all the code suggestions you had and rebased. This should be now good to go.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 93917` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>icon-slicer</li>
<li>openzone-cursors</li>
</ul>
</details>

File: pkgs/tools/X11/icon-slicer/default.nix
Comment (SuperSandro2000) [inline pkgs/tools/X11/icon-slicer/default.nix:18]:
```suggestion
  nativeBuildsInputs = [ popt pkg-config ]

  buildInputs = [ gdk-pixbuf ];
```

File: pkgs/tools/X11/icon-slicer/default.nix
Comment (SuperSandro2000) [inline pkgs/tools/X11/icon-slicer/default.nix:13]: Please add a comment why we need this.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 93917` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>icon-slicer</li>
<li>openzone-cursors</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 93917` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>icon-slicer</li>
<li>openzone-cursors</li>
</ul>
</details>

**Pull request merged**
